docs/dev/devel-env.rst

```diff
@@ -111,6 +111,17 @@ Update the strings with your client ID and secret. After doing this, your
 ListenBrainz development environment is able to authenticate and log in from
 your MusicBrainz login.
 
+Also, in order for last.fm import to work, you should also update your last.fm api key in this file. Look for the following section in the file.
```

Comment:
We try to write `Last.FM` instead of `last.fm` in documentation.

Reply:
Ok :+1: